src/lib/executeSourceTransfer.js

```diff
@@ -2,6 +2,12 @@
 
 const log = require('../common').log.create('executeSourceTransfer')
 const validator = require('./validate')
+const promiseRetry = require('promise-retry')
+const retryOpts = { // retry 10 times within 5 seconds
+  retries: 10,
+  factor: 1,
+  minTimeout: 500
+}
```

Comment:
promise-retry will only start the retry once the promise has been rejected (meaning the ledger already got the fulfillment call and the connector got the error message back)